lsp/java/src/main/java/com/itsaky/androidide/lsp/java/debug/LspThreadInfo.kt

```
@@ -65,7 +67,30 @@ class JavaStackFrame(
 						frame.location() == location
 					}?.run {
 						val variables = mutableListOf<AbstractJavaVariable<*>>()
-						val thisObject = this.thisObject()
+
+						try {
+							logger.info("QA TEST: Waiting 5s... KILL THE APP!")
+							Thread.sleep(5000)
+						} catch (e: Exception) {}
```

detekt (1.23.8)

[warning] Empty catch block detected. If the exception can be safely ignored, name the exception according to one of the exemptions as per the configuration of this rule.

(detekt.empty-blocks.EmptyCatchBlock)